docs/2101UserGuide.md

```diff
@@ -216,84 +216,103 @@ Examples:
 <hr class="feature-class-separator">
 
 ### View commands
+In CCACommander, you can quickly change the view to fit your current use case, be it to check for the enrolment of an event, 
+filtering for a member and more!
 
-#### List all Members and all Events : `list`
+#### View Events of Member : `viewMember`
+
+If you want to view the events in which a particular member is enrolled, you can do so via the `viewMember` command.
+
+Here’s how you can use it:
+`viewMember MEMBER_INDEX`
 
-List all members and all events in the CCA in two separate columns.
+Here is an example command where we want to view the events in which the **member** at index 3 in our member list is enrolled:
```

Comment:
I think index should be bolded instead of member, cause we generally bold the parameters in the other commands
```suggestion
Here is an example command where we want to view the events in which the member at **index** 3 in our member list is enrolled:
```